compiler/damlc/lib/DA/Cli/Damlc/Packaging.hs

```diff
@@ -81,6 +81,20 @@ createProjectPackageDb opts thisSdkVer deps dataDeps = do
     createDirectoryIfMissing True $ dbPath </> "package.conf.d"
 
     deps <- expandSdkPackages (filter (`notElem` basePackages) deps)
+
+    let versionize name = case name of
+          "daml-script" -> name <> "-" <> sdkVersion
+          "daml-trigger" -> name <> "-" <> sdkVersion
+          _ -> name
+
+    let units =
+          [ GHC.stringToUnitId (versionize base)
+          | dep <- deps
```

Comment:
This also needs to take `dataDeps` into account.

Reply:
yes, it does now